tests/volta/test_determine_platform.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
```

Comment:
this test file as well as `tests/volta/test_build_url.py` may as well be removed and you should cover it with actually calling build_url in `tests/volta/test_download_and_unpack_archive.py`

i think with unit tests and mocking its easy to make a bunch of tests and i'd like to steer more towards quality over quantity